### ISSUE TITLE
Timeout waiting for validation

### ISSUE THREAD

**Issue by gunstr** [esphome/esphome-vscode#96]
I have installed the esphome-vscode extension and configured it for validation against a dashboard exposed at localhost:6052. The dashboard works as it should when opened from a browser. The connetion from the vscode extension to the dashboard is established when I open a file set to ESPhome language but it seems the validation does not return anything. This is the log output I see when I do a first edit of the file:

`Configuring ESPHome with dashboard validation at localhost:6052...
Using ESPHome dashboard at: ws://localhost:6052/vscode server: http://localhost:6052/
Connection established.
Validating file:///d%3A/OneDrive/Documents/Dator/HomeAssistant/configSkene/ESPhome/test_temp_2.yaml
send {"type":"validate","file":"d:\\OneDrive\\Documents\\Dator\\HomeAssistant\\configSkene\\ESPhome\\test_temp_2.yaml"}
Timeout waiting for previous validation to complete. Discarding.`

Any suggestion how to resolve this is appreciated.

**Comment by glmnet** [comment]
yes, validation is broken and is expected to be fixed with the next esphome release 2025.04 i guess scheduled for today. Please reopen if you still have issues.
